test/scripts/scripts.test.js

```diff
@@ -31,10 +38,11 @@ describe('Decorating', () => {
   });
 
   it('Loads lana.js upon calling lana.log the first time', async () => {
+    await waitFor(() => window.lana?.loaded);
```

Comment:
This is only working due to a bug in the test - lana.log hasn't been called explicitly.  It is called when catching a ReferenceError thrown by the martech script.  That needs to be fixed/removed so that this test runs correctly.

I think we can remove the `loaded` property, and instead do a reference check after calling lana the first time (once above is fixed):
```
expect(window.lana.log).to.exist;
const initialLana = window.lana.log;
sinon.spy(console, 'log');
await window.lana.log('test', { clientId: 'myclient', sampleRate: 0 });
await waitFor(() => initialLana !== window.lana.log);
```

I will look into the martech issue tomorrow.

Reply:
OK. So you will fix the martech issue before we can update this?